lang/ruby/lib/avro/io.rb

```diff
@@ -376,13 +376,16 @@ def read_union(writers_schema, readers_schema, decoder)
         read_data(selected_writers_schema, readers_schema, decoder)
       end
 
+      def read_record_data(field, readers_field, decoder)
```

Comment:
What about renaming this function `read_field_data`? It doesn't read all of a record's data, just the field.